test/test_bernese.py

```diff
@@ -0,0 +1,138 @@
+"""
+Kør Bernese fillæser med testdata
+"""
+import math
+from pathlib import Path
+
+from fire.io.bernese.bernese import BerneseSolution, Kovarians
+
+
+def test_read():
+    """
+    Indlæs og test output fra tre konkrete sæt med forskellige værdier og sektionsformater
+    """
+
+    # Datasæt 1 - første sæt, uden eksisterende COV-fil eller spredning
+
+    reader1 = BerneseSolution(
+        addneq_fil=Path(
+            "test/data/ADDNEQ2_1273"  # denne fil har ingen spredningssektion
+        ),
+        crd_fil=Path("test/data/COMB1273.CRD"),
+        cov_fil=Path(
+            "test/data/COMB1273.COV"
+        ),  # denne fil findes ikke (og det er OK med COV-filer)
```

Comment:
Hvis filen ikke findes bør der så ikke smides en exception?

Reply:
Det kunne vi i princippet godt - lave en NoCOVFileException, som man så kan håndtere ved at læse de andre værdier. ER det en farbar vej fremfor at man tester for om Kovarians er None og så læser a_posteriori_RMS i stedet?

Reply:
Er det ikke smartest at være eksplicit og fejle med en `FileNotFoundError` og så lade brugeren løse problemet? I det her tilfælde kan problemet løses på to måder: 1) giv den rigtige sti eller 2) lad være med at pege på en cov-fil. Ved 1) skal covarianserne selvfølgelig læses fra cov-filen og ved 2) må vi så lade `BerneseSolution` falde tilbage på det information vi kan udlede af ADDNEQ- og CRD-filerne.

Reply:
Så noget i denne stil?

```Python
        # Dernæst tilføj kovariansmatricer fra COV-fil hvis den eksisterer
        if cov_fil is not None:
            if not cov_fil.exists():
                raise FileNotFoundError
            with open(cov_fil, "r") as cov:
                self.cov_parse(cov.readlines())
```

Reply:
Ja, det må være noget i den stil. Du kan evt. specificere fejlen yderligere med `raise FileNotFoundErrror(f"{cov_fil} ikke fundet")` eller noget i den stil så man har en ide om hvor det går galt.

Reply:
Det er committet. Testcasen er tilsvarende ændret til at angive `None` i stedet for ikke-eksisterende fil.